concepts/switch-statement/about.md

```diff
@@ -145,6 +145,23 @@ Starting with Java 14 (available as a preview before in Java 12 and 13) it is po
    However if you use the new `->` notation it must be followed by either: a single statement/expression, a `throw` statement or a `{}` block.
    No more confusion!
 
+The first LTS (Long Term Support) version that had these improvements was Java 17, released on September, 2021.
+
+In addition, a feature called `Guarded Patterns` was added, which allows you to do checks in the case label itself.
```

Comment:
I recommend mentioning the Java version in this sentence to make it clear we're now talking about a feature added in Java 21.

```suggestion
In addition, a feature called `Guarded Patterns` was added in Java 21, which allows you to do checks in the case label itself.
```